jumpscale/packages/tfgrid_solutions/chats/exposed.py

```diff
@@ -46,6 +46,8 @@ def solution_type(self):
 
     @chatflow_step(title="Solution to be exposed")
     def exposed_solution(self):
```

Comment:
can we check the length of all solutions in the beginning of the chatflow (and show only a list of what we have solutions for)